cmd/filegen/main.go

```diff
@@ -556,17 +550,8 @@ func isMintValueValid(mintValue string) error {
 	return nil
 }
 
-func getNodesKeyGen(consensusType string) crypto.KeyGenerator {
-	var suite crypto.Suite
-
-	switch consensusType {
-	case "bls":
-		suite = kyber.NewSuitePairingBn256()
-	case "bn":
-		suite = kyber.NewBlakeSHA256Ed25519()
-	default:
-		suite = nil
-	}
+func getNodesKeyGen() crypto.KeyGenerator {
+	suite := kyber.NewSuitePairingBn256()
```

Comment:
please rebase as this was changed on master

Reply:
done